### ISSUE TITLE
Create Credits page

### ISSUE THREAD

**Issue by ashlingam** [hackforla/website#571]
### Overview
Create credits page for website. This page will house attributions for all images, graphics, and photographs used.

### Action Items
- [x] Content strategy
   - [x] Research attribution requirements for different image sources on the web. Identify pricing and attribution placement requirements. 
   - [ ] Purchase subscription (if appropriate) to image source web site
- [x] UI/UX
  - [x] Create mockups for desktop and mobile
  - [x] Revise mockups based on feedback
  - [x] Hand off to dev
- [ ] Development
  - [ ] Implment new Credits page
  - [ ] Insert attributions for all images on Credits page

### Resources/Instructions
https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1074%3A0

**Comment by yuikomajima** [comment]
Noun Project guidelines: https://thenounproject.zendesk.com/hc/en-us/articles/200509928-How-do-I-give-creators-credit-in-my-work-

**Comment by yuikomajima** [comment]
Freepik guidelines: https://support.freepik.com/hc/en-us/articles/208976585-Attribution-How-when-and-where-

**Comment by yuikomajima** [comment]
Credits page mocked up here: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1074%3A1

If this looks OK, we can ask Stella and Danielle too add any images/icons they have used into this page as well. (Or have them share me a link and I can input)

**Comment by ExperimentsInHonesty** [comment]
@yuikomajima i was imagining it would actually have a small version of each icon or image on it for easy reference

**Comment by yuikomajima** [comment]
Here is a page with images included: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1129%3A1159

**Comment by ExperimentsInHonesty** [comment]
@yuikomajima - Agreed all the designers should add their icons to this first version of the page @leestella748 @DanielekPark

This mockup looks great!  
I think we could add a section at the top (the usual white box that goes all the way across where we say something about Why it's important to give credit and if anyone finds content on the site that needs attribution, please let us know by writing to [need to provide an appropriate email address or form here]

**Comment by yuikomajima** [comment]
@ExperimentsInHonesty I added the white hero with illustration. Would be great if we can add some better copy, if you are able to provide!

**Comment by leestella748** [comment]
@yuikomajima I just added my icons on the side of the mockup and have a few comments/questions. 

1.  It would be great if we had a uniform size for all icons on the credits page.
2. I wasn't sure why the vector images didn't link directly to the Freepik page, so I added the links for the images I used.  Feel free to unlink them if we don't need to.
3. We could group the icons together either by page (ie. Getting Started, Hack Nights, Projects) or by source (ie. Noun Project, Iconfinder, Freepik) which could potentially reduce the page length as well.

**Comment by ExperimentsInHonesty** [comment]
Decision made at team meeting was to create a new format for the info:

Icon Name:
Where Used:
Provider (tbd on the name):
Artist:

Note for developers: for the markdown file, we will also need image location

**Comment by yuikomajima** [comment]
I made all of the changes above. 

In regards to the new text format I made it the following: 

Content (since we might be using not only icons and imagery but maybe videos etc. in the future)
Used in 
Artist
Provider

figma file: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1129%3A1159

Please lmk if we need any changes!
I took a pass at the text in the hero, but please feel free to assign a content person to do the copy.

**Comment by ExperimentsInHonesty** [comment]
@yuikomajima Can you change the name from Content to Name?

**Comment by yuikomajima** [comment]
Name has been changed.

**Comment by ExperimentsInHonesty** [comment]
This is ready for development.   Great Job @yuikomajima

**Comment by ghost** [comment]
**Progress:** Working through the update: certain questions increased progress time (will message PM)
**Blocks:** Unclarity 
**Availability:** Sunday, Monday, Tuesday
**ETA:** Sunday (earliest), Tuesday (latest)

**Comment by ExperimentsInHonesty** [comment]
@ethanh256 
markdown files in the _credits folder will have the following
Content (since we might be using not only icons and imagery but maybe videos etc. in the future)
Used in:
Artist:
Provider:
image url:
type: {icon, resource}
display: {false}

what to create:
collection will be called _credits
include file will be called /includes/credit-item.html
edit _config.yml to include the credits in the collection

**Comment by ethanh256** [comment]
**Progress:** almost finished
**Blocks:** Time
**Availability:** Sunday
**ETA:** Sunday

**Comment by ghost** [comment]
Progress: Finished the design
Blocks: Div does not shape well on smaller screens
Availability: This week
ETA: This week

**Comment by ghost** [comment]
Done, ready to push

**Comment by ghost** [comment]
<img width="1438" alt="Screen Shot 2020-08-16 at 10 47 37 AM" src="https://user-images.githubusercontent.com/58455740/90340573-fe7a8500-dfad-11ea-9ed2-40bdfa4da344.png">
<img width="1434" alt="Screen Shot 2020-08-16 at 10 48 51 AM" src="https://user-images.githubusercontent.com/58455740/90340586-14884580-dfae-11ea-8cf3-141368db12b7.png">
<img width="1440" alt="Screen Shot 2020-08-16 at 10 49 05 AM" src="https://user-images.githubusercontent.com/58455740/90340587-1c47ea00-dfae-11ea-855b-89ebe842498e.png">

**Comment by ghost** [comment]
Waiting for review

**Comment by ghost** [comment]
Working with Josh

**Comment by ghost** [comment]
After a long period of time of not doing anything, I've actually made some pretty good progress today. 

Progress: I cleaned up most of the SCSS file, started on fixing the responsiveness
Blocks: I have to admit, I'm not a huge fan of CSS so I've been putting this off. Sorry about that.
ETA: Today
Availability: Today

**Comment by ghost** [comment]
New pull request: #721 

**Credits Page:** The page is now responsive to screen size changes, has no duplicate class names from other pages (all tags are custom). Made the HTML more semantic. 

Examples of responsiveness of webpage on various screen sizes:

![credits](https://user-images.githubusercontent.com/58455740/92339964-739d1f80-f06d-11ea-81c5-5de3b1cd08d0.png)
![credits medium](https://user-images.githubusercontent.com/58455740/92339965-7566e300-f06d-11ea-99bc-b582de7ed671.png)
![credits mobile](https://user-images.githubusercontent.com/58455740/92339970-7730a680-f06d-11ea-823d-6e28f9a79881.png)

@jbubar Helped me, thanks!
